extensions/investspec/investspec.py

```diff
@@ -1,18 +1,19 @@
 import docutils
+import gettext
 import importlib
 
 import pint
-from natcap.invest import spec_utils
+from natcap.invest import install_language, spec_utils
 
 INPUT_TYPES_HTML_FILE = 'input_types.html'
 # accepted geometries for a vector will be displayed in this order
 GEOMETRY_ORDER = [
-    'POINT',
-    'MULTIPOINT',
-    'LINESTRING',
-    'MULTILINESTRING',
-    'POLYGON',
-    'MULTIPOLYGON']
+    _('POINT'),
+    _('MULTIPOINT'),
+    _('LINESTRING'),
+    _('MULTILINESTRING'),
+    _('POLYGON'),
+    _('MULTIPOLYGON')]
```

Comment:
Ah, interesting. Certainly a lot to keep track of!

Reply:
Oh, yeah. I wish there weren't so many separate translation mechanisms going on! Maybe a simpler way will become clear once we have used it a bit.